src/pytest_flyte/plugin.py

```diff
@@ -2,16 +2,41 @@
 import os
 import pathlib
 import shutil
+import subprocess
 from contextlib import contextmanager
+from pathlib import Path
 
 import pytest
+import yaml
 from flytekit.clients import friendly
-from pytest_docker.plugin import DockerComposeExecutor
-
 from jinja2 import Environment, FileSystemLoader
+from pytest_docker.plugin import DockerComposeExecutor
 
 PROJECT_ROOT = os.path.dirname(__file__)
-TEMPLATE_ENV = Environment(loader=FileSystemLoader(os.path.join(PROJECT_ROOT, "templates")))
+TEMPLATE_ENV = Environment(
+    loader=FileSystemLoader(os.path.join(PROJECT_ROOT, "templates"))
+)
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--local",
+        action="store",
+        default=True,
+        help="Local/Remote",
+    )
+    parser.addoption(
+        "--flyte-platform-url",
+        action="store",
+        default=None,
+        help="Flyte Platform URL",
+    )
+    parser.addoption(
+        "--proto-path",
```

Comment:
`--proto-path`  name is not generic?  Currently, we have two types of path 
1. host machine path
2. tar.gz URL path